Assets/HoloToolkit-UnitTests/Editor/Utilities/WorldAnchorManagerTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using UnityEngine;
+using NUnit.Framework;
+using HoloToolkit.Unity.Tests.Extensions;
+
+namespace HoloToolkit.Unity.Tests
+{
+    public class WorldAnchorManagerTests
+    {
+        [SetUp]
+        public void ClearScene()
+        {
+            TestUtils.ClearScene();
+            var managers = new GameObject("Managers");
+            managers.AddComponent<WorldAnchorManager>();
```

Comment:
This won't be needed with the other changes above however, you could make this work with:
```chsarp
managers.CallAwake().CallStart();
```
Explanation [here](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/master/Assets/HoloToolkit-UnitTests#2-any-monobehaviour-methods-such-as-awake-start-and-update-will-not-be-called-by-unity-in-tests)

Reply:
I read the documentation, but I didn't understand when to use these methods. Thanks for the example.